ddcz/models/used/users.py

```diff
@@ -103,6 +157,20 @@ def profile_url(self):
             kwargs={"user_profile_id": self.pk, "nick_slug": self.slug},
         )
 
+    @property
+    def registration_date(self):
```

Comment:
I'd argue that this should either have `_str` suffix or be handed in the template. When I am retrieving `something_date` from model, I expect a date/time-like object, IMHO. 

I'd keep it consistent as with creations—we also use a filter there.

Reply:
I used the date templatetag then and I am returning the object. :)